src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionIntegrationTest.java

```diff
@@ -1271,7 +1271,7 @@ public void happyValidAuthKey() {
   @Order(5)
   class CreateCollectionWithEmbeddingServiceTestParameters {
     @Test
-    public void failNotProvideRequiredParameters() {
+    public void failNotProvideRequiredProviderParameters() {
```

Comment:
+1 for renaming to better reflect what is being tested!

Reply:
Sure, change to `failWithMissingRequiredProviderParameters`